superset/models/helpers.py

```diff
@@ -424,7 +425,10 @@ class ExtraJSONMixin:
     def extra(self) -> Dict[str, Any]:
         try:
             return json.loads(self.extra_json)
-        except Exception:  # pylint: disable=broad-except
+        except (TypeError, JSONDecodeError) as exc:
+            logger.error(
+                "Unable to load an extra json: %r. Leaving empty.", exc, exc_info=True
```

Comment:
Can `exc` or `exc_info` log data from `self.extra_json` ?

Reply:
Good idea! I will add it to an exception logger.

Reply:
maybe I did not explain myself right, sorry. I want to make sure that `self.extra_json` never gets logged by `TypeError` or `JSONDecodeError` by `exc` or `exc_info`

Reply:
@dpgaspar do you mind to take a look again? I have one doubt: do you think is it possible there could be raised exception while accessing `self.extra_json` for logger?

Reply:
Och, I see, then I just did the opposite thing! :)

Reply:
Just to be sure I check that and any of them won't show what `self.extra_json` contains.
`JSONDecodeError` should something like:
`json.decoder.JSONDecodeError: Unterminated string starting at: line 1 column 11 (char 10)`
with stacktrace.
And `TypeError` something like this for example:
`TypeError: the JSON object must be str, bytes or bytearray, not dict`

Reply:
nice then, thks